x-pack/plugins/spaces/public/copy_saved_objects_to_space/copy_saved_objects_to_space_action.tsx

```diff
@@ -18,12 +18,12 @@ export class CopyToSpaceSavedObjectsManagementAction extends SavedObjectsManagem
 
   public euiAction = {
     name: i18n.translate('xpack.spaces.management.copyToSpace.actionTitle', {
-      defaultMessage: 'Copy to space',
+      defaultMessage: 'Make a copy',
```

Comment:
@mdefazio had the same comment above (on a screenshot, not this line of code). I changed it to match his newer mockups, if my memory serves correctly it's because we wanted to better differentiate between the Copy and Share actions. I'm happy to change it back if that's the consensus, though.

Reply:
I changed it back to "Copy to space".